en-US/Conductors_Companion/A_Appendix/A2_Adventure_Ideas_Generators/Core/AIG_Treasure.md

```diff
@@ -18,7 +18,7 @@ Roll a `d10` to determine monetary treasure, or a `d20` for monetary and magical
 |     11     | Potion of Healing      |
 |     12     | Other potion           |
 |     13     | Scroll or spell scroll |
-|     14     | Bag of Holding         |
+|     14     | _<span class="item item-Bag_of_Holding_bag_of_holding">bag of holding</span>_ |
```

Comment:
Note: This will be updated with a link, once we have the item _bag of holding_ in the Conductor's Companion.